src/routers/client/index.ts

```diff
@@ -13,5 +14,6 @@ router.use("/event", eventRouter);
 router.use("/users", userRouter);
 router.use("/auth", authRouter);
 router.use("/enrollments", tokenValidationMiddleware, enrollmentRouter);
+router.use("/hotels", hotelRouter);
```

Comment:
A reserva de hotéis é uma rota protegida, senti falta do middleware de validação do token.